### DISCUSSION TITLE
Only about 15% of plugins found

### DISCUSSION THREAD

**Discussion by belovedmike** [apohl79/audiogridder#731] [Help]
Server version: 1.2.0 (BETA 5)
Hardware: M1 Mac Mini
OS: Big Sur 11.5.2

Running the server on my local machine for the first time to try it out and I'm only getting about 15% of my plugins to be active in the plugins list. It can't seem to find any plugins from Plugin Alliance, Slate Audio, Soundtoys, iZotope, NI, IK, or most of my AA plugins. Obviously I expect the UAD ones to fail but why would they all fail? See attached video.

Shouldn't need to say it but all are 100% legit plugins that I own the license for (licenses stored on my iLok if that makes a difference)

So far I've tried trashing the config file and opened AG server again but I get the same result.

Looking at the Scan logs for any of the failed plugins I just see this:
```
[21:25:22.642|app|138751d08] commandline: -scan "/Library/Audio/Plug-Ins/VST/Plugin Alliance/Black Box Analog Design HG-2.vst|VST"
[21:25:22.643|app|138751d08] scan mode: format=VST id=/Library/Audio/Plug-Ins/VST/Plugin Alliance/Black Box Analog Design HG-2.vst
[21:25:22.643|server|1048f17b0] scanning id=/Library/Audio/Plug-Ins/VST/Plugin Alliance/Black Box Analog Design HG-2.vst fmt=VST
[21:25:22.648|app|138751d08] ...failed
[21:25:22.648|app|138751d08] initialise complete
```

I tried downgrading to the stable version 1.1.1 and get exactly the same issue
What steps can I take to get these running? Any help appreciated. Thanks!

https://user-images.githubusercontent.com/91316343/134735681-820d0cdf-f93f-430d-85bd-cca193b96169.mp4

**Comment by belovedmike** [comment]
[Logs.zip](https://github.com/apohl79/audiogridder/files/7230909/Logs.zip)
Attaching logs. The server logs show "skipping" next to most plugins

**Comment by apohl79** [comment]
The server skips already scanned plugins. Choose wipe and scan to trigger a rescan of all plugins.

I don't know why scans were failing for you. You need to make sure the licenses are available on the server machine, but you said you did.

**Reply by belovedmike** [reply]
I've tried that sadly: rescanned, uninstalled and reinstalled, downgraded to previous version, same result.
The only thing I can think is that it's having difficulty with licenses stored on an iLok, can anyone confirm? I'll move a couple over to the Mac itself and see if they appear. Will report back.

Anybody else had difficulty with so many failed plugins before?

**Reply by apohl79** [reply]
I never had issues with iLok.

**Comment by CaBuffalo** [comment]
No, I don't have any issue with iLok neither.

On Mon, Sep 27, 2021 at 2:40 AM Andreas Pohl ***@***.***>
wrote:

> I never had issues with iLok.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/731#discussioncomment-1389354>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HBYQXMI4XDWHMHIX3UEA3XXANCNFSM5EWWTCXA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by belovedmike** [comment]
Seems to be a problem specifically with VSTs. If I install the AU version of the plugin (for example iZotope RX 8 Standard) then this shows up fine in AG, but I usually only use the VST3 or 2 version. I don't really want to have to go back and install AU versions of all of my plugins (if they even exist) - is this a known issue or am I alone in this? 

Anyone got RX 8 loading as a VST in AG? Same goes for all of the Plugin Alliance plugins - AU loads, VST2 or 3 don't
